pkg/utils/version.go

```diff
@@ -0,0 +1,29 @@
+package utils
+
+import (
+	"context"
+
+	semverlib "github.com/Masterminds/semver/v3"
+	"github.com/google/go-github/v45/github"
+)
+
+func CheckLatestVersion(currentVersion string) {
+	client := github.NewClient(nil)
+	latestRelease, _, err := client.Repositories.GetLatestRelease(context.Background(), "pluralsh", "plural-cli")
+	if err == nil {
+		latestVersion := *latestRelease.TagName
+		cv, err := semverlib.NewVersion(currentVersion)
+		if err != nil {
+			return
+		}
+		lv, err := semverlib.NewVersion(latestVersion)
+		if err != nil {
+			return
+		}
+		if cv.Major() == lv.Major() && cv.Minor() == lv.Minor() && cv.Patch() >= lv.Patch() {
```

Comment:
Could we just not check the patch version at all?  Trying to think of any case where it'd be ahead of the gh version, but seems like basically equivalent with a bit more coverage

Reply:
the patch version is the last number from version v0.5.20 -> 20 so it does matter

Reply:
oh wait I read this backwards, this makes more sense now